### PR TITLE
added new touch point form tab

### PR DESCRIPTION
**Pull request by ilodidoj**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/733)

## What does this change?
This allow users to start a feedback form on modal from primary concern page

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Re-check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/templates/forms/report_primary_complaint.html [hunk 1]
```diff
@@ -31,4 +31,7 @@
 {% block page_js %}
   {{ block.super }}
   <script src="{% static 'js/primary_complaint.js' %}"></script>
+  <script src="https://touchpoints.app.cloud.gov/touchpoints/e0517981/js" async></script>
+  <script src="{% static 'js/modal.js' %}"></script>
+  <script src="{% static 'js/redirect-modal.js' %}"></script>
```
Comment (cantsin) [inline crt_portal/cts_forms/templates/forms/report_primary_complaint.html:36]: We don't need the `js/modal.js` or the `js/redirect-modal.js` included here. These are unrelated to touchpoints.

**Review by cantsin — CHANGES_REQUESTED** [review]
I think we just need that one script, not the modal javascript code.

File: crt_portal/cts_forms/templates/forms/report_primary_complaint.html (hunk 1 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/forms/report_primary_complaint.html]: @cantsin Removed all modal js files. Please review and merge.

**Review by cantsin — APPROVED** [review]
LGTM
